### DISCUSSION TITLE
Sign in users automatically without intermediate step

### DISCUSSION THREAD

**Discussion by TheOnlyTails** [nextauthjs/next-auth#10670] [Help]
Is there a way, when using the SvelteKit Auth.js adapter, to automatically redirect users to a specific provider's sign in page when loading the page, preferably on the server?
Something like this:
```ts
// src/routes/(auth)/signin/+page.server.ts
export const load = async () => {
	signIn("github");
}
```

**Comment by JasperAlexander** [comment]
I think you need to use middleware, but I don't know how that works with SvelteKit.
